catalogs/hive-metastore-common/src/main/java/org/apache/gravitino/hive/converter/HiveDataTypeConverter.java

```diff
@@ -116,7 +117,17 @@ public TypeInfo fromGravitino(Type type) {
         Types.StructType structType = (Types.StructType) type;
         List<TypeInfo> typeInfos =
             Arrays.stream(structType.fields())
-                .map(t -> fromGravitino(t.type()))
+                .map(
+                    t -> {
+                      // Hive does not support comments in struct fields. This is no way to preserve
+                      // comments in struct fields when converting from Gravitino to Hive.
+                      // See: https://issues.apache.org/jira/browse/HIVE-26593
+                      Preconditions.checkArgument(
+                          t.comment() == null,
+                          "Hive does not support comments in struct fields: %s",
+                          t.name());
+                      return fromGravitino(t.type());
+                    })
```

Comment:
Compared to throwing an exception. In my opinion, it is better to add a warning log and ignore the comment field for Hive table. What do you think?

Directly throwing an exception and failing the request seems too strict.

Reply:
Can you please also update the related docs?